### PR TITLE
Feature/SOF-7252 feat: Add tutorial for ZSL notebook

### PR DESCRIPTION
**Pull request by VsevolodX**

### CODE REVIEW

File: lang/en/docs/materials-designer/header-menu/input-output/standata-import.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Import from Standata
+
+Materials Designer can source structural data from the Standata materials dataset. This tutorial explains how to import Graphene and Ni from Standata into the Materials Designer session.
+
+## Select "Import from Standata" Option
```
Comment (VsevolodX) [inline lang/en/docs/materials-designer/header-menu/input-output/standata-import.md:5]: 1. Import

File: lang/en/docs/materials-designer/header-menu/input-output/standata-import.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Import from Standata
+
+Materials Designer can source structural data from the Standata materials dataset. This tutorial explains how to import Graphene and Ni from Standata into the Materials Designer session.
```
Comment (VsevolodX) [inline lang/en/docs/materials-designer/header-menu/input-output/standata-import.md:3]: Add link to standata

File: lang/en/docs/materials-designer/header-menu/input-output/standata-import.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Select materials, for example, 'Graphene' and 'Ni' from the dropdown list.
+
+<img src="/images/materials-designer/import/import_from_standata.png" alt="Import Gr and Ni from Standata"/>
+
+Click 'Submit' to import these materials into the Materials Designer session.
+## View Imported Items
+
+Materials (Graphene and Ni in this case) should now be available in the materials list.
+
+<img src="/images/materials-designer/import/graphene_and_ni_imported.png" alt="Gr and Ni available in materials list"/>
```
Comment (VsevolodX) [inline lang/en/docs/materials-designer/header-menu/input-output/standata-import.md:22]: Add Links section

File: mkdocs.yml [hunk 4]
```diff
@@ -173,7 +173,9 @@ nav:
             - Combinatorial Sets:                    tutorials/materials/combinatorial-screening.md
             - Interpolated Sets:                     tutorials/materials/interpolated-sets.md
             - Molecule on a Surface:                 tutorials/materials/molecule-surface.md
-            - Interface:                             tutorials/materials/slabs-interface.md
+            - Interface with 3D editor:              tutorials/materials/slabs-interface.md
+            - Interface with Python:                 tutorials/materials/interface-with-python.md
+            - Interface with minimal strain:         tutorials/materials/jupyterlite-zsl.md
```
Comment (timurbazhirov) [inline mkdocs.yml:178]: - Interface quick setup (3D Editor)
- Interface, no strain minimization (Python Transformation)
- Interface, minimal strain (JupyterLite Session)

**Review by timurbazhirov — APPROVED** [review]

**Pull request merged**